src/api/index.js

```diff
@@ -1,6 +1,7 @@
+import axios from 'axios';
+
 const URL = 'http://34.213.11.120/organizations/1/users'
 
 export default () => {
-    return fetch (URL) 
-        .then(response => Promise.all([response, response.json()]))
+  return axios.get(URL).then(response => response)
```

Comment:
Here you are using a class called API instead of using controllers. What do you think about having a controller for each resource that you'll be fetching from the backend? Similar to what you are using for the user in the UserController. For example, if you are fetching Commerces, you'll have a `CommercesController` where you'll do every request related to the commerces. Having something called `APIController` is confusing because it will grow way too much and will be dificult to mantain afterwards. What do you think?